portal-api/api-tests/helpers/convertToTimestamp.api-test.js

```diff
@@ -0,0 +1,13 @@
+const convertToTimestamp = require('../../src/v1/helpers/convertToTimestamp');
+
+describe('convertToTimestamp()', () => {
+  it('Should return a timestamp in the normal time format for date passed as a string', () => {
```

Comment:
Need some more tests cases please for substanstive test coverage, could you please introduce following test scenarios:

1. Null or empty `timestamp` provided.
2. Epoch provided instead of string.

I understand helper function is being called after various UI level validation (so above arguments are edge cases) but a safety net in a helper function improves code quality and reliability.

Reply:
done